dataverse-webapp/src/main/webapp/dataverse_header.xhtml

```diff
@@ -160,11 +160,13 @@
                                 <ul class="dropdown-menu language-select">
                                     <ui:repeat value="#{settingsWrapper.configuredLocales.entrySet().toArray()}" var="language">
                                         <li jsf:rendered="#{language.value ne dataverseSession.localeTitle}">
-                                            <h:commandLink value="#{language.value}"
-                                                            actionListener="#{dataverseSession.updateLocaleInViewRootAndRedirect(language.key)}"
-                                                            lang="#{language.key}"
-                                                            style="#{'background: url(/resources/images/flags/'.concat(language.key).concat('.svg) no-repeat 90% center; background-size: 35px 35px')}">
-                                            </h:commandLink>
+                                            <p:commandLink value="#{language.value}"
+                                                           lang="#{language.key}"
```

Comment:
I don't think that primefaces supports lang attribute. We can check it by inspecting rendered html we probably won't see  `lang` being defined. Probably we should use passthrough attribute here (`pt:lang`)